### PR TITLE
VACMS 18800 - remove recordEvent and related variables

### PR DESCRIPTION
**Pull request by eselkin**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

- Added disableAnalytics on the va-button due to recordEvent being triggered with extra variable data. 
- Sitewide team

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18800

## Testing done

- Both standard DS Analytics and recordEvent were being triggered on click. Changed to only recordEvent
- Manual tested on facility locator VA health search and other types.
- Check that on search, moving map, clicking button that appears on map that GA default push from DS component does not trigger but recordEvent does.

## Screenshots

No display or accessibility change on component
<details>
<summary>console from PR GA events</summary>

</details>

<details>
<summary>console from current GA events</summary>
Screenshot 2024-08-05 at 2 47 23 PM

</details>

## What areas of the site does it impact?

Facility locator (button on map area)

## Acceptance criteria

Duplicate event listed in description was removed

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added (no visible or a11y change)
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (unauthenticated)

## Requested Feedback

In RI - go to /find-locations
Search for some address/Zip code/State
Select some type of facility
Move the map after search
See that button appears for search in new area
Open console in dev tools (make sure you have Adswerve running - if not start Adswerve and rerun above steps)
Click button and see that gtm.cta-button-click does not run but fl-search does

### CODE REVIEW

**Review by randimays — COMMENTED** [review]
Waiting for review instance, but it would be helpful if you include screenshot(s) of the GA events from the console in the PR description for posterity.

File: src/applications/facility-locator/components/SearchAreaControl.jsx [hunk 1]
```diff
@@ -37,13 +37,14 @@ const SearchAreaControl = ({
...
       <va-button
-        uswds
+        uswds="true"
         id="search-area-control"
         className={buttonClass}
         text={buttonLabel()}
         disabled={!isEnabled}
         ariaLive="assertive"
         onClick={handleClick}
+        disable-analytics="true"
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/applications/facility-locator/components/SearchAreaControl.jsx:47]: I just know it didn't work as disableAnalytics. I'll try without the "true".

File: src/applications/facility-locator/components/SearchAreaControl.jsx [hunk 2]
```diff
@@ -37,13 +37,14 @@ const SearchAreaControl = ({
   return (
     <div id="search-area-control-container" className={containerClass()}>
       <va-button
-        uswds
+        uswds="true"
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/applications/facility-locator/components/SearchAreaControl.jsx:40]: Removed

File: src/applications/facility-locator/components/SearchAreaControl.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (eselkin) [reply src/applications/facility-locator/components/SearchAreaControl.jsx]: Removed `="true"`

**Review by randimays — APPROVED** [review]
LGTM!

**Pull request merged**